src/main/webapp/app/pages/companyManagement/CompanyDetailsPage.tsx

```diff
@@ -34,7 +34,7 @@ export default class CompanyDetailsPage extends React.Component {
       id: 'companyUserCount',
       Header: <span className={styles.tableHeader}>Users</span>,
       minWidth: 60,
-      accessor: 'companyUserCount',
+      accessor: (data: CompanyDTO) => data.numberOfUsers,
```

Comment:
This is minor, you can change to `numberOfUsers` instead of calling the method.

Reply:
Thanks, also will change the accessor for companyName to `name` instead of calling the method.